src/api/app/models/workflow.rb

```diff
@@ -5,20 +5,63 @@ def initialize(workflow:, scm_extractor_payload:, token:)
     @workflow = workflow
     @scm_extractor_payload = scm_extractor_payload
     @token = token
+    @unsupported_steps = extract_unsupported_steps
```

Comment:
The instance variable and the method can have the same name:

```suggestion
    @unsupported_steps = unsupported_steps
```